pkg/monitor/sqsevent/sqs-monitor_test.go

```diff
@@ -377,7 +377,7 @@ func TestMonitor_EC2NoInstances(t *testing.T) {
 	}
 }
 
-func TestMonitor_EC2NoDNSName(t *testing.T) {
+func TestMonitor_EC2NoDNSNameFailure(t *testing.T) {
```

Comment:
Not sure we should change the name here since the result is no error. With "Failure" in the name I would expect `err != nil`

Reply:
fine with me